docs/examples/tensorflow/demo/nvutils/image_processing.py

```diff
@@ -98,8 +98,8 @@ def __init__(self,
             output_dtype=dali.types.FLOAT,
             crop=(height, width),
             image_type=dali.types.RGB,
-            mean=[121., 115., 100.],
-            std=[70., 68., 71.],
+            mean=[0.485 * 255,0.456 * 255,0.406 * 255],
+            std=[0.229 * 255,0.224 * 255,0.225 * 255]),
```

Comment:
Could you explain why this happend? I thought that these constants were specific to PyTorch pretrained models

Reply:
Leftover from other things, reverting.